prime-angular-frontend/src/app/modules/site-registration/pages/technical-support/technical-support.component.html

```diff
@@ -7,37 +7,15 @@
                                [form]="form"
                                (submit)="onSubmit()">
 
-    <app-same-as selectFor="technicalSupport"></app-same-as>
+    <app-same-as selectFor="technicalSupport"
+                 (selected)="onSelect($event)"></app-same-as>
 
     <ng-container appPageSubheaderSummary>
-      This individual may be either a staff person at the site or an independent IT contractor. They will be contacted
-      in the event of a technology-related issue (e.g. malware, problematic network traffic).
-
-      <br>
-      <br>
-
-      The Technical Support Contact should be familiar with:
-
-      <ul>
-        <li>
-          details of the practice's network structure;
-        </li>
-        <li>
-          internet connections in use at the site;
-        </li>
-        <li>
-          the application(s) used to access PharmaNet at the site; and
-        </li>
-        <li>
-          security mechanisms such as firewalls and anti-virus measures (both generally and those in use at the site.)
-        </li>
-      </ul>
-
-      This individual should be able to quickly provide a network diagram of the site, including peripherals, wireless
-      access points, connected devices, and all applications using the internet router(s). This individual may be the
-      same as the Signing Authority, the Administrator of PharmaNet Onboarding, or the Privacy Officer; however, they
-      must also possess the technical skill and understanding to answer complex network- and application-related
-      questions.
+      Technical Support Contact A site staff person, corporate IT or independent IT contractor, familiar with the
```

Comment:
"Technical Support Contact A site staff person" - this reads weird. Should the "A" be capitalized?

Reply:
Haha probably not, but this text was copy-pasted from provided text.